packages/react-examples/src/react/PeoplePicker/PeoplePicker.List.Example.tsx

```diff
@@ -156,6 +182,28 @@ export const PeoplePickerListExample: React.FunctionComponent = () => {
         onChange={onToggleDelayResultsChange}
         styles={checkboxStyles}
       />
+      <Checkbox
+        label="Required People Picker"
+        checked={isPickerRequired}
+        // eslint-disable-next-line react/jsx-no-bind
+        onChange={onRequiredButtonClick}
+        styles={checkboxStyles}
+      />
+      <Checkbox
+        label="Show Label"
+        checked={showPickerLabel}
+        // eslint-disable-next-line react/jsx-no-bind
+        onChange={onShowLabelButtonClick}
+        styles={checkboxStyles}
+      />
+      {showPickerLabel && (
+        <TextField
+          label={'People Picker Label'}
+          value={pickerLabel}
+          // eslint-disable-next-line react/jsx-no-bind
+          onChange={onPickerLabelChanged}
```

Comment:
In case you go with the suggestion above

```suggestion
          onChange={onPickerLabelChange}
```